test/lifecycle_test.dart

```diff
@@ -682,11 +682,7 @@ external List getUpdatingSetStateLifeCycleCalls();
 external List getNonUpdatingSetStateLifeCycleCalls();
 
 /// A test helper to record lifecycle calls
-abstract class LifecycleTestHelper {
-  Map context;
```

Comment:
This was causing analysis errors since it was overriding the context setter without `covariant`.